### ISSUE TITLE
How to write answer for maths questions 

### ISSUE THREAD

**Issue by manishamittal16** [tecnickcom/tcexam#331]

**Comment by nicolaasuni** [comment]
Are you asking for the students to be able to write math formulas on their answers?

**Comment by manishamittal16** [comment]
yes

**Comment by andiplusplus** [comment]
I think we need file upload form element on test taker side

**Comment by manishamittal16** [comment]
Yes,we need option for upload subjective answer file from user side.

**Comment by mamans86** [comment]
> Yes,we need option for upload subjective answer file from user side.

for a while you can use my fork of TCExam, download link https://github.com/mamans86/tcexam-mobile-friendly/releases/tag/20.07.19
i have added an ability to upload multimedia files on textarea.

**Comment by manishamittal16** [comment]
Okay thank you so much ,I will try it

**Comment by manishamittal16** [comment]
I checked it,but don't understand how admin evaluate it and from where he/she found uploaded ffiles except cache folder

**Comment by mamans86** [comment]
Admin can see the image uploaded by student from Test -> Evaluation, and evaluate from there.

**Comment by manishamittal16** [comment]
I already checked it... But there, it is only display the  path of file...

**Comment by mamans86** [comment]
What system do you use? I run on localhost and shared hosting, and both are fine. The image displayed as expected.

**Comment by mamans86** [comment]
i'm sorry, after double check, there's still mistake on that feature. i'm forgot something, and i'll fix it again as soon as posible.

**Comment by manishamittal16** [comment]
Okay,sure let me know when you fixed it.
And I tried on localhost as well as on GoDaddy server.

**Comment by mamans86** [comment]
you can try to replace these 2 files:
https://github.com/mamans86/tcexam-mobile-friendly/commit/ac0a343a0869cec6334e4572771ab57987542c76
.
i hope it will fix the problem

**Comment by manishamittal16** [comment]
Okay
I will try it.

**Comment by mamans86** [comment]
to improve user experience when test taker adding file to editor, you can replace these 4 files  :
- shared/jscripts/main.js (https://raw.githubusercontent.com/mamans86/tcexam-mobile-friendly/8dee989111a5b3a72dd19544a001c8749a079655/shared/jscripts/main.js)
- public/code/tce_select_userphoto.php (https://raw.githubusercontent.com/mamans86/tcexam-mobile-friendly/8dee989111a5b3a72dd19544a001c8749a079655/public/code/tce_select_userphoto.php)
- public/styles/default.css (https://raw.githubusercontent.com/mamans86/tcexam-mobile-friendly/8dee989111a5b3a72dd19544a001c8749a079655/public/styles/default.css)
- public/styles/default_rtl.css (https://raw.githubusercontent.com/mamans86/tcexam-mobile-friendly/8dee989111a5b3a72dd19544a001c8749a079655/public/styles/default_rtl.css)

**Comment by manishamittal16** [comment]
![answer](https://user-images.githubusercontent.com/64191398/88250424-37804c00-ccc5-11ea-89e8-0736d55465e1.PNG)
Please check this image.It only display path,after making changes in 2 files

**Comment by mamans86** [comment]
those 2 files need javascript enable on your browser. Or, please try using Google Chrome browser

**Comment by manishamittal16** [comment]
Now working fine,thank you so much.

**Comment by manishamittal16** [comment]
According to my requirement, I have changed in the maximum files 'TCEXAM', but now i want to add upload media file option in my existing updated file, so there are any possibility to do this changes in the existing files ? I need all files path if is it possible.
Thank You

**Comment by nicolaasuni** [comment]
TCExam supports mathematical notation via LaTeX and MathML in question/answer text — see doc/LATEX.md for enabling LaTeX rendering. Closing as answered/usage.
